release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -772,6 +772,10 @@ For more information about this more flexible SR-IOV network deployment during N
 ==== Update to HAProxy 2.6
 With this release, {product-title} is updated to HAProxy 2.6.
 
+[id="ocp-4-14-max-log-length-sidecar"]
+==== Support for configuring the maximum length with sidecar logging in the Ingress Controller
+Previously, the maximum length of the syslog message in the Ingress Controller was 1024 bytes. Now, the maximum value can be increased.
+For more information, see xref:../networking/ingress-operator.adoc#nw-configure-ingress-access-logging_configuring-ingress[Allowing IngressController to modify the HAProxy log length when using a sidecar].
```

Comment:
For the link text, would suggest using either the name of the heading it links to
```suggestion
For more information, see xref:../networking/ingress-operator.adoc#nw-configure-ingress-access-logging_configuring-ingress[Configuring Ingress access logging].
```

or the title of the section you want.
```suggestion
For more information, see xref:../networking/ingress-operator.adoc#nw-configure-ingress-access-logging_configuring-ingress[Allow the Ingress Controller to modify the HAProxy log length when using a sidecar].
```